### PR TITLE
DOCS-3515 Added Link to RTF doc

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/asm-secret-group-concept.adoc [hunk 1]
```diff
@@ -5,33 +5,40 @@ endif::[]
...
 
 == Availability and Consistency
 
-By implementing Edit Sessions every time a user needs to edit secrets inside a secret group, Secrets Manager guarantees service of a consistent set of secrets.
+Secrets Manager implements edit sessions every time a user needs to edit secrets inside a secret group to guarantee service of a consistent set of secrets.
 
-Consider the scenario where you need to renew secrets inside a secret group: You need to click the Edit button in the Anypoint Platform UI. This opens an edit session during which the secret group locks out other users, and you can edit all the necessary security objects inside the secret group. While the edit session is open, Secrets Manager serves the same secrets it had before you opened the edit session. This guarantees that any redeployed application would still see the same, consistent secret group even though you are modifying it.
+For example, if you need to renew secrets inside a secret group, you click the *Edit* button in the Secrets Manager UI. This opens an edit session and locks out other users while you edit the necessary security objects inside the secret group.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-concept.adoc]: I'd start with an intro statement first to set context, something like "Secrets Manager ensures that operations are not interrupted while you are modifying secrets. When you edit a secrets group, other users are locked out while you..."

File: modules/ROOT/pages/asm-secret-group-concept.adoc [hunk 2]
```diff
@@ -5,33 +5,40 @@ endif::[]
...
 
 image::asm-secret-group-concept-4876b.png[]
 
-If the editing user forgets about an open edit session and is not available, you can click the Cancel Edit button to clear the that user's session. Note that this action deletes all that user's changes, but it allows you to click the now available Edit button to open a new edit session.
+If the user who is editing the secret group leaves the edit session open and is not available to cancel the edit session, you can click the *Cancel Edit* button to clear that user's session. 
+
+[NOTE]
+If you cancel a user's edit, it deletes all that user's changes. 
+
+The *Edit* button is then available for you to open a new edit session.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-concept.adoc:30]: Just an overall note that I'm surprised to see this level of detail in a conceptual topic.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 3]
```diff
@@ -21,7 +21,7 @@ The name of your secret group must:
 * Start with a letter
 * Contain only letters, numbers, and dashes (but cannot end with a dash)
 
-The secret group appears in the *Secret Groups* list view. Now you can add the necessary secret types.
+The secret group appears in the *Secret Groups* list view. You can now add the necessary secret types, for example, a keystore or a truststore.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: What's the actual action? "...list view. Select a listed item (click a listed item) to add the necessary..." ?

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 4]
```diff
@@ -140,13 +145,19 @@ Add a list of certificates that were revoked by certificate authorities before t
 
 == Add a TLS Context
 
+To include a certificate-key pair in a TLS context to encrypt inbound traffic to Runtime Fabric or Mule 4 based proxies, you must enable the TLS context.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Do you need "based" here?

**Review by Mulesoft-Mysti — APPROVED** [review]
Great that you added context setting lead ins to those tasks, really improves ease-of-use/understanding! I made a few suggestions, none are showstoppers.

File: modules/ROOT/pages/asm-secret-group-concept.adoc (hunk 2 above)
Comment (valkyrie69) [reply modules/ROOT/pages/asm-secret-group-concept.adoc:30]: Yeah, that's the way I inherited it. I would have made this a task.

**Pull request merged**